src/components/Keyword/KeywordList/index.js

```diff
@@ -0,0 +1,329 @@
+import React,{useCallback,useEffect,useState,useRef} from 'react';
+import KeywordHeader from '../KeywordHeader';
+import * as S from './styles';
+import { AUNURI,AOUMIR } from 'constant';
+import { useSelector, useDispatch } from 'react-redux';
+import { getKeywordList,deleteKeywordList,moveKeywordItem,readKeywordItem } from 'store/keyword';
+import { getTitle,makeDeleteQuery } from '../utills';
+
+const MENU_ITEM = [
+    {   
+        id:0,
+        title:'전체'
+    },
+    {   
+        id:1,
+        title:'아우누리'
+    },
+    {   
+        id:2,
+        title:'아우미르'
+    },
+    {   
+        id:3,
+        title:'대신 전해드립니다-koreatech'
+    },
+];
+
+const KeywordList = () => {
+
+    const userInfo = useSelector((state)=>state.auth);
+    const {
+        keywordList,
+        deleteKeywordListResponse,
+        readKeywordItemResponse,
+        getKeywordListResponse} = useSelector((state)=>state.keyword);
+    const dispatch = useDispatch();
+
+    const [menu,setMenu] = useState('전체');
+    const [list,setList] = useState([]);
+    const [checkAll,setCheckAll] = useState(false);
+    const [checkListId,setCheckListId] = useState([]);
+    const [readNotification, setReadNotification] = useState(false);
+    const [notReadNotification, setNotReadNotification] = useState(false);
```

Comment:
이건 하나의 값으로 관리할 수 있지 않을까요...?
`notRead~~` 는 되게 어색해보이네요

```suggestion
    const [readNotification, setReadNotification] = useState(false);
```